requirements.txt

```
@@ -1,2 +1,3 @@
 pip<24.0
 immutabledict<2.2.6
+typing-extensions<4.8
```

```suggestion
typing-extensions>=4.8,<5
```
as suggested on the homepage.

---

It worked! I was too pessimistic. Let's see how dependabot deals with the update